packages/core/ai/src/tools/tool-resolver-service.ts

```diff
@@ -5,27 +5,24 @@
 import { type AiTool, AiToolkit } from '@effect/ai';
 import { Context, Effect, Layer } from 'effect';
 
-import { AiToolNotFoundError } from '../errors';
-
 import { type ToolId } from './tool';
 
 export class ToolResolverService extends Context.Tag('@dxos/ai/ToolResolverService')<
   ToolResolverService,
   {
-    readonly resolve: (id: ToolId) => Effect.Effect<AiTool.Any, AiToolNotFoundError>;
+    readonly resolve: (id: ToolId) => Effect.Effect<AiTool.Any | void>;
```

Comment:
hmm? why void? if its optional I'd prefer the prior `Effect.Effect<AiTool.Any, AiToolNotFoundError>`